assets/app/scripts/directives/logViewer.js

```diff
@@ -233,16 +233,8 @@ angular.module('openshiftConsole')
             APIDiscovery
               .getLoggingURL()
               .then(function(url) {
-                // TODO: update lodash for _.get, would rather do this:
-                // var projectName = _.get($scope, 'context', 'project', 'metadata', 'name');
-                // var containerName = _.get($scope, 'options', 'container');
-                var projectName = $scope.context &&
-                                  $scope.context.project &&
-                                  $scope.context.project.metadata &&
-                                  $scope.context.project.metadata.name;
-
-                var containerName = $scope.options &&
-                                    $scope.options.container;
+                var projectName = _.get($scope.context, ['project', 'metadata', 'name']);
+                var containerName = _.get($scope.options, 'container');
```

Comment:
Yup, that will work.  Not sure why I opted to do it that way here... the array lets you pass in a var, but isn't needed in this case.  I'll update.